test/helper.rb

```diff
@@ -21,7 +22,7 @@ module ARHPTestSetup
   private
 
   def arhp_create_models
-    return if Object.const_defined?("Test1")
+    return if Object.const_defined?('Test1')
```

Comment:
hoho ... I meant `quote wars` ... but actually the quotes are also unnecessary here :)